server/fishtest/static/css/live_elo.css

```diff
@@ -1,30 +0,0 @@
-div.error {
-  display: none;
-}
```

Comment:
We could also remove all `error` related code right?

https://github.com/glinscott/fishtest/blob/2dc3970bf45ad5c3415e2e658cffed9e1d7b4d3c/server/fishtest/templates/tests_live_elo.mak#L25

https://github.com/glinscott/fishtest/blob/2dc3970bf45ad5c3415e2e658cffed9e1d7b4d3c/server/fishtest/static/js/live_elo.js#L146

Reply:
I think so.

Reply:
Correct, the alert_() function was dropped in #1397
DEV updated.